symfony/assets/scripts/class/Storage.ts

```diff
@@ -36,6 +36,15 @@ export default class Storage {
     this.save(key, value ? "1" : "0", expireSeconds);
   }
 
+  public static remove(key: string): void {
+    try {
+      localStorage.removeItem(this.dataPath(key));
+      localStorage.removeItem(this.expiresPath(key));
+    } catch (e) {
+      console.log("Failed to remove data.", e);
```

Comment:
Another question from a frontend-inexperienced person. I know that this was moved, but it made me notice that I've used `log` here. Should we switch to `error`/`warn`/`info` instead?

Reply:
It wouldn't hurt, but I don't think it's a huge problem. For smaller projects where your console volume is relatively low anyways, I don't know that it matters as much in JS as it does on the terminal, where `log` goes to stdout and `error` goes to stderr and maybe you're piping them to different locations.

It's probably more worth investing in a replacement/wrapper for `console`. Typically, you'll want to have these kinds of logs on your development builds but then have them noop or be removed by the build process for prod, since otherwise this is all happening for the customer too. That's usually what frontend JS developers more focus on.